packages/skills/weather_station/behaviours.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This package contains a scaffold of a behaviour."""
+
+from aea.protocols.oef.message import OEFMessage
+from aea.protocols.oef.models import Description
+from aea.protocols.oef.serialization import OEFSerializer, DEFAULT_OEF
+from aea.skills.base import Behaviour
+from .weather_station_dataModel import WEATHER_STATION_DATAMODEL
```

Comment:
Please check develop for how imports work now

Reply:
Fixed that. It was the PyCharm errors that I was talking about.